tests/unit/metadata/test_metadata_upgrader.py

```diff
@@ -0,0 +1,763 @@
+
+
+def test__upgrade_constraints_positive_and_negative():
+    """Test the ``_upgrade_constraints`` method with ``Positive`` and ``Negative`` constraints.
+
+    Input:
+        - Old metadata dict with the following constraints:
+        - ``Positive`` constraint with one column and 'strict' as True.
+        - ``Positive`` constraint with one column and 'strict' as False.
+        - ``Positive`` constraint with multiple columns and 'strict' as True.
+        - ``Positive`` constraint with multiple columns and 'strict' as False.
+        - ``Negative`` constraint with one column and 'strict' as True.
+        - ``Negative`` constraint with one column and 'strict' as False.
+        - ``Negative`` constraint with multiple columns and 'strict' as True.
+        - ``Negative`` constraint with multiple columns and 'strict' as False.
+
+    Output:
+        - Metadata with the following constraints:
+        - ``Positive`` constraint for every ``Positive`` that had strict as True.
+        - ``ScalarInequality`` for every ``Positive`` that had strict as False.
+        - ``Negative`` constraint for every ``Negative`` that had strict as True.
+        - ``ScalarInequality`` for every ``Negative`` that had strict as False.
+    """
+    # Setup
+    old_constraints = [
+        {
+            'constraint': 'sdv.constraints.tabular.Positive',
+            'columns': 'a',
+            'strict': True
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Positive',
+            'columns': ['b', 'c'],
+            'strict': True
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Positive',
+            'columns': 'd',
+            'strict': False
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Positive',
+            'columns': ['e', 'f'],
+            'strict': False
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Negative',
+            'columns': 'a',
+            'strict': True
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Negative',
+            'columns': ['b', 'c'],
+            'strict': True
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Negative',
+            'columns': 'd',
+            'strict': False
+        },
+        {
+            'constraint': 'sdv.constraints.tabular.Negative',
+            'columns': ['e', 'f'],
+            'strict': False
+        },
+    ]
+    old_metadata = {'constraints': old_constraints}
+
+    # Run
+    new_constraints = _upgrade_constraints(old_metadata)
+
+    # Assert
+    expected_constraints = [
+        {
+            'constraint_name': 'Positive',
+            'column_name': 'a'
+        },
+        {
+            'constraint_name': 'Positive',
+            'column_name': 'b'
+        },
+        {
+            'constraint_name': 'Positive',
+            'column_name': 'c'
+        },
+        {
+            'constraint_name': 'ScalarInequality',
```

Comment:
Why can't we pass the `strict` parameter to the Positive/Negative constraints in the new metadata?